src/symspell.rs

```diff
@@ -109,6 +114,54 @@ impl<T: StringStrategy> SymSpell<T> {
         }
         true
     }
+    /// Load multiple bigram entries from a file of bigram/frequency count pairs.
+    ///
+    /// # Arguments
+    ///
+    /// * `corpus` - The path+filename of the file.
+    /// * `term_index` - The column position of the word.
+    /// * `count_index` - The column position of the frequency count.
+    /// * `separator` - Separator between word and frequency
+    pub fn load_bigram_dictionary(
+        &mut self,
+        corpus: &str,
+        term_index: i64,
+        count_index: i64,
+        separator: &str,
+    ) -> bool {
+        if !Path::new(corpus).exists() {
+            return false;
+        }
+
+        let file = File::open(corpus).expect("file not found");
+        let sr = BufReader::new(file);
+        let line_parts_len = if separator == " " { 3 } else { 2 };
+        for (i, line) in sr.lines().enumerate() {
+            if i % 50_000 == 0 {
+                println!("progress: {}", i);
+            }
+            let line_str = line.unwrap();
+            let line_parts: Vec<&str> = line_str.split(separator).collect();
+            if line_parts.len() >= line_parts_len {
```

Comment:
It would be handy for interoperation with Javascript bindings if this method could be splitted into two parts along the lines of `load_dictionary` or if the direct I/O could be avoided completely

Reply:
I agree with @MRoci 

@sai-prasanna any chance you can do it?

Reply:
I split it and added wasm binding for loading bigram dictionary.